sunpy/map/tests/test_mapbase.py

```diff
@@ -1452,13 +1451,33 @@ def test_submap_inputs(generic_map2, coords):
         smap = generic_map2.submap(*args, **kwargs)
         assert u.allclose(smap.dimensions, (3, 3) * u.pix)
 
+def test_contour_deprecation_warning(simple_map):
+    with pytest.warns(SunpyDeprecationWarning, match="The 'contour' method is deprecated and will be removed in a future release. Please use 'get_contours' instead."):
+        simple_map.get_contours([1.5, 2.5])
```

Comment:
```suggestion
        simple_map.contour([1.5, 2.5])
```

Reply:
I get I have made a mistake here, but I usually do check the tests. I'm not sure why they weren't failing locally- my repository is up-to-date.

Reply:
It isn't about making a mistake, that's normal. There are plenty of PRs of mine here that were and are riddled with mistakes. 

It's just good practice to check if changes do not break the relevant tests.